frontend/src/components/widgets/CameraInput/styled-components.ts

```diff
@@ -78,7 +79,7 @@ export interface StyledBoxProps {
 export const StyledBox = styled.div<StyledBoxProps>(({ theme, width }) => ({
   backgroundColor: theme.colors.secondaryBg,
   borderRadius: `${theme.radii.md} ${theme.radii.md} 0 0`,
-  width,
+  width: "100%",
```

Comment:
Hm, I'm not sure what the effect of setting `width: 100%` rather than to the given parameter is.

I'm guessing that in most cases the `height` parameter being set based on `width` will give us the desired result, but it's worth verifying there aren't any weird cases

Reply:
We need this to keep consistent video/button sizes during resizing
here is a video for old behavior:

https://user-images.githubusercontent.com/6664805/205316289-30859e0c-cfc3-4489-bf6e-cdd409c40ea1.mov

And this new behavior for `width: "100%"`:

https://user-images.githubusercontent.com/6664805/205316450-d644f8ae-c4be-4965-8609-0931e9744082.mov